player.lua

```diff
@@ -4,6 +4,10 @@ function mineunit:get_players()
 	return players
 end
 
+-- this one belongs to player_api mod
+-- format player_attached[player_name] = boolean/nil
+_G.player_api = { player_attached = {} }
+
```

Comment:
If this is from mod then that mod should probably be loaded for tests instead of adding it here.

Here's simple example for checking out from git, loading mod and executing some tests with multiple loaded mods: https://github.com/mt-mods/wrench/pull/5

For very simple stuff I've also used fixtures a lot, pipeworks, digilines and mtg examples:
https://github.com/mt-mods/technic/blob/master/technic_cnc/spec/fixtures/pipeworks.lua
https://github.com/mt-mods/technic/blob/master/technic/spec/fixtures/digilines.lua
https://github.com/mt-mods/technic/blob/master/technic/spec/fixtures/default.lua

If significant enough and included in mineunit then it should at least be separate module that could be loaded with something like `mineunit("extras/player_api")`.
While https://github.com/S-S-X/mineunit/tree/master/default is not in very good shape it gives simple example: `mineunit("default/functions")`

Reply:
So far I've been using simple fixture https://github.com/SwissalpS/spectator_mode/blob/invite/spec/fixtures/player_api.lua

That table was the only one I've come accross from that mod, so I'll remove it from mineunit. It is simple to just copy a fixture file to next mod that might use it.

Reply:
IOW: I didn't think it merited an extra file in mineunit or I wouldn't have tried to piggy-back it into player module in the first place.

Reply:
Reason behind this is fairly simple: mineunit is testing framework and therefore it _should_ crash and report error if anything that is not available in core is used without explicitly defining it.
Including some commonly used fixtures / modules would be fine and could be even done through separate project / repository (I could maybe even extend core version selector to include installer for custom global fixtures/modules).

`default` is something I should get rid of (removing all autoloaders for it), I guess it might be what gave you bad example and is actually unwanted behavior. Currently `default` is incorrectly loaded and made available with `mineunit("core")`.